monty/io.py

```diff
@@ -17,7 +17,7 @@
 
 
 def zopen(filename: Union[str, Path], *args, **kwargs) -> IO:
-    r"""
```

Comment:
I guess this is a typo and some accidential remaining from previous docstring changes?